solr/test-framework/src/java/org/apache/solr/cloud/ZkTestServer.java

```diff
@@ -113,7 +112,7 @@
   class ZKServerMain {
 
     private volatile ServerCnxnFactory cnxnFactory;
-    private volatile ZooKeeperServer zooKeeperServer;
+    public volatile ZooKeeperServer zooKeeperServer;
```

Comment:
minor: seems it could stay private?
```suggestion
    private volatile ZooKeeperServer zooKeeperServer;
```